ci-operator/step-registry/rosa/sts/account-roles/create/rosa-sts-account-roles-create-commands.sh

```
@@ -104,16 +91,46 @@ if [[ ! -z "$PERMISSIONS_BOUNDARY" ]]; then
    PERMISSIONS_BOUNDARY_SWITCH="--permissions-boundary ${PERMISSIONS_BOUNDARY}"
 fi
 
-# Whatever the account roles with the prefix exist or not, do creation.
+# Create account roles, falling back to latest available version if target lacks published policies
 echo "Create the ${CLUSTER_SWITCH} account roles with the prefix '${ACCOUNT_ROLES_PREFIX}'"
 echo "rosa create account-roles -y --mode auto --prefix ${ACCOUNT_ROLES_PREFIX} ${CLUSTER_SWITCH} ${VERSION_SWITCH} ${ARN_PATH_SWITCH}"
+
+create_ret=0
 rosa create account-roles -y --mode auto \
                           --prefix ${ACCOUNT_ROLES_PREFIX} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)